src/libraries/System.Security.Cryptography.Pkcs/tests/Certificates.cs

```diff
@@ -32,6 +34,13 @@ internal static class Certificates
         public static readonly CertLoader RsaOaep2048_Sha1Parameters = new CertLoaderFromRawData(RawData.RsaOaep2048_Sha1ParametersCert, RawData.RsaOaep2048_Sha1ParametersPfx, "1111");
         public static readonly CertLoader RsaOaep2048_Sha256Parameters = new CertLoaderFromRawData(RawData.RsaOaep2048_Sha256ParametersCert, RawData.RsaOaep2048_Sha256ParametersPfx, "1111");
         public static readonly CertLoader RsaOaep2048_NoParameters = new CertLoaderFromRawData(RawData.RsaOaep2048_NoParametersCert, RawData.RsaOaep2048_NoParametersPfx, "1111");
+        public static readonly CertLoader SlhDsaSha2_128s_Ietf = new CertLoaderFromRawData(SlhDsaTestData.IetfSlhDsaSha2_128sCertificate, SlhDsaTestData.IetfSlhDsaSha2_128sCertificatePfx, "PLACEHOLDER");
+        public static readonly CertLoader[] SlhDsaGeneratedCerts = LoadSlhDsaCerts();
+
+        private static CertLoader[] LoadSlhDsaCerts() =>
+            SlhDsaTestData.GeneratedKeyInfosRaw
+                .Select(cert => new CertLoaderFromRawData(cert.Certificate, cert.SelfSignedCertificatePfx, cert.EncryptionPassword))
```

Comment:
The GeneratedKeyInfosRaw value isn't a cert, so presumably `cert` should be renamed to `info`.